src/main/resources/db/migration/create-ddl.sql

```diff
@@ -0,0 +1,50 @@
+create table address (id bigint generated by default as identity, city varchar(255), state varchar(255), street varchar(255), type varchar(255), zip varchar(255), primary key (id));
```

Comment:
why does this file exist as a src folder at the root? it should be in a dedicated SQL folder in the root